docs/api-docs/provider-apis/tax/overview.mdx

```diff
@@ -9,8 +9,7 @@ The [Tax Provider API](/docs/rest-contracts/tax) allows you to provide business-
 
 The [Tax Provider API](/docs/rest-contracts/tax) works in conjunction with a BigCommerce app, so you will need to [build an app](/docs/integrations/apps) that integrates the [Tax Provider API](/docs/rest-contracts/tax).
 
-Multi-tenant tax providers can publish their BigCommerce app so that it's discoverable by anyone or publish their app as unlisted so it can only be installed using a URL. Furthermore, tax solutions that have been built in-house or for specific merchants by digital agencies are also supported as private instances and will only work on the specified stores.
-
+You can publish your BigCommerce app so that it's discoverable by anyone or publish their app as unlisted so it can only be installed using a URL.
```

Comment:
seems to be mixed context here , your vs their

Reply:
Good call, thanks!